src/app/reactions-details/reactions-image/reactions-image.component.scss

```diff
@@ -0,0 +1,3 @@
+:host {
```

Comment:
this will render the component without wrapping tag, so instead of:

```
<reacitons-image-component>
  <img ... />
<reacitons-image-component>
```

will be just

```
<img />
```